src/devices/tuya.ts

```diff
@@ -13467,6 +13467,82 @@ const definitions: DefinitionWithExtend[] = [
+    		e.numeric('entry_sensitivity', ea.STATE_SET).withValueMin(10).withValueMax(100).withValueStep(10).withUnit('%')
+    			.withDescription('Sensitivity threshold triggered for the first time when the target enters the detection range'),
+    		e.numeric('entry_distance', ea.STATE_SET).withValueMin(0).withValueMax(8).withValueStep(0.1).withUnit('m')
+    			.withDescription('Dectection distance when unoccupied'),
+    		e.numeric('entry_filter_time', ea.STATE_SET).withValueMin(0).withValueMax(0.5).withValueStep(0.05).withUnit('s')
+    			.withDescription('Sensitivity threshold triggered for the first time when the target enters the detection range '),
+    		e.numeric('departure_delay', ea.STATE_SET).withValueMin(5).withValueMax(7200).withValueStep(1).withUnit('s')
+    			.withDescription('Confirmation time after the target disappears'),
+    		e.numeric('block_time', ea.STATE_SET).withValueMin(0).withValueMax(10).withValueStep(0.1).withUnit('s')
+    			.withDescription('Time for the target to be detected again after switching from manned(occupy) to unmanned(unoccupy) mode'),
+    		e.enum('status_indication', ea.STATE_SET, ['OFF', 'ON'])
+    			.withDescription('Indicator light will turn on when human presence is detected'),
+    		e.enum('sensor', ea.STATE_SET, ['on', 'occupied', 'unoccupied'])
+    			.withDescription(`The radar sensor can be set in three states: on, occupied and unoccupied. For example, if set to occupied, ` +
+    				`it will continue to maintain presence regardless of whether someone is present or not. If set to unoccupied, the unoccupied ` +
+    				`state will be maintained permanently.`),			
+    		e.enum('scene_preset', ea.STATE_SET, ['Custom', 'Toilet', 'Kitchen', 'Hallway', 'Bedroom', 'Livingroom', 'Meetingroom', 'Factory default'])
+    			.withDescription(`Presets`),
+    		e.enum('distance_report_mode', ea.STATE_SET, ['Normal', 'Occupancy detection'])
+    			.withDescription('Indicator light will turn on when human presence is detected'),			
+    		e.enum('debug_mode', ea.STATE_SET, ['OFF', 'ON'])
+    			.withDescription(`In debug mode, radar will report more information, can be used to identify interference`),
+    		e.numeric('debug_distance', ea.STATE).withDescription('Real time distance to target').withUnit('m'),
+    		e.numeric('debug_countdown', ea.STATE).withDescription('Time before the target disappears').withUnit('s'),
+    
+    	],
+    	meta: {            
+    		tuyaSendCommand: 'sendData',
+    		tuyaDatapoints: [
+    //			[1, 'presence', tuya.valueConverter.trueFalse1],
+    			[101, 'entry_sensitivity', tuya.valueConverter.raw],
+    			[102, 'entry_distance', tuya.valueConverter.divideBy100],
+    			[103, 'departure_delay', tuya.valueConverter.raw],
+    			[104, 'entry_filter_time', tuya.valueConverter.divideBy100],
+    			[105, 'block_time', tuya.valueConverter.divideBy10],
+    			[107, 'illuminance_lux', tuya.valueConverter.divideBy10],
```

Comment:
```suggestion
    			[107, 'illuminance', tuya.valueConverter.divideBy10],
```

And use `e.illuminance()` instead.